### PR TITLE
[Feature][29659] Show tags

### PR DESCRIPTION
**Pull request by ntbai**
https://edu-redmine.sun-asterisk.vn/issues/29659

### CODE REVIEW

**Comment by ntbai** [comment]
ready

File: app/controllers/boards_controller.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@ def create
 
   def show
     @labels = @board.labels
+    @lists = @board.lists.where(closed: false)
```
Comment (saobv-2223) [inline app/controllers/boards_controller.rb]: viết scope nhé

**Comment by ntbai** [comment]
ready

**Comment by ntbai** [comment]
ready

**Pull request merged**